### ISSUE TITLE
Screen starts panning by itself, after setting a favorite

### ISSUE THREAD

**Issue by Rowin63** [osmandapp/OsmAnd#17225]
### Description
It happend now already a few times: 
after setting a favorite (not really everytime, but almost), the screen starts panning by itself (map moves to the right / I´m going west). Can´t stop this movement, but it´s gone after closing and re-open the app. 

Happens on two different devices, also in the 4.5.0#54m

### Steps to reproduce
select a point at the map and set it as a favorite

### Actual result
screen rolls away 

### Expected result
don´t do this :-)

### Your Environment
OsmAnd Version: +4.5.0 2023-05-12
Android/iOS version: 13
Device model: S20plus

**Comment by Rowin63** [comment]
That's pretty crazy. Now I set more than 30 favorites, and it did not happen at all... 🤔

**Comment by Rowin63** [comment]
Two days now I tried to force this issue again without "success". Setting lots of favorites without this happening again.
I doubted my perception...
But then, in a moment when I did not expect it, here it was again!

See the attached video (this is release 4.5.0#60m)

After setting a favorite, screen is moving and moving and moving... 
Even minimizing (hiding) the app and bring it back does not stop the scrolling. Only closing stops it.

https://github.com/osmandapp/OsmAnd/assets/95540936/e5013bb0-44a8-4f9c-8560-3679941e0249

I can´t upload a longer clip, I can do this (center the map) again and again and the screen keeps on moving and moving.
